_sass/template/print-pdf.scss

```diff
@@ -215,8 +215,10 @@ $frontmatter-reference-style: lower-roman !default; // lower-roman, decimal, see
 // using $spaced-paras.
 $floated-element-classes: sidenote, qr-code !default;
 
-// Other PDF features
+// Title page
 $title-page-logo-width: $paragraph-indent * 3 !default;
+$title-page-logo-font: $font-text-main !default;
+$title-page-logo-font-size: $font-size-default - 1 !default;
```

Comment:
Does this work if $font-size-default has a unit like pt or rem, and the (-1) has no unit?

Reply:
@LouiseSteward Yes, a unitless number in Sass is compatible with any number with a unit. [Demo here](https://sass-lang.com/playground/#eJwzNFTJK81NSi2yUjA0KKhQ0FUwtObiSspPqVSo5lJQSMvPK9EtzqxKtVKAqrPmquUCAJwdD6U=).